settings.gradle.kts

```diff
@@ -1,3 +1,10 @@
 rootProject.name = "kotlin-result"
 
 include("example")
+
+pluginManagement {
+    repositories {
+        maven("https://dl.bintray.com/kotlin/kotlinx" )
+        gradlePluginPortal()
+    }
+}
```

Comment:
Is this stuff necessary or can we get away with just the repository declarations in the `build.gradle.kts` files?

Reply:
seems to be the default way to configure repository location for gradle plugins now.
Gradle actually specifies that pluginManagement _must_ be specified inside settings.gradle